.tav.yml

```
@@ -385,11 +385,17 @@ tedious-v12-v15:
   # first and last majors subset of '12.x || 13.x || 14.x'
   versions: '12.0.0 || 12.3.0 || 13.0.0 || 13.2.0 || 14.0.0 || 14.7.0 || >14.7.0 <15'
   commands: node test/instrumentation/modules/tedious.test.js
-tedious:
+tedious-v15-v16:
   name: tedious
   node: '>=14'
   versions: '15.0.0 || 15.1.3 || >15.1.3 <16' # first and last majors subset of '15.x'
   commands: node test/instrumentation/modules/tedious.test.js
+tedious:
+  name: tedious
+  node: '>=15'
```

```suggestion
  node: '>=16'
```

Tedious' package.json has:

```
  "engines": {
    "node": ">=16"
  },
```